Cargo.toml

```diff
@@ -31,3 +32,6 @@ opt-level = 3
 lto = "thin"
 incremental = false
 codegen-units = 16
+
+[patch.crates-io]
+pasta_curves = { git = "https://github.com/lurk-lab/pasta_curves", branch = "dev" }
```

Comment:
This should no longer be necessary after https://github.com/lurk-lab/arecibo/pull/333